internal/plugins/ansible/templates/config/kustomize/auth_proxy_patch.go

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Could you please keep the common files in the same directory? Has no need to move them and in order to verify this one, for example, I'd like to able to diff helm/ansible plugins dir and get just its differences.

Reply:
I'm leaving the 1 big comment on the structure discussion open, but resolving all the little ones to make comments easier to parse.